src/run_part.F90

```diff
@@ -569,6 +601,12 @@ subroutine spec_file_read_run_part(file, run_part_opt, aero_data, &
           call spec_file_open(sub_filename, sub_file)
           call spec_file_read_aero_data(sub_file, aero_data)
           call spec_file_close(sub_file)
+       ! FIXME: Temporary to run PartMC. Replace with initialization from TChem
```

Comment:
PartMC can not get the aerosol data from TChem yet. Therefore we read in aero_data as in a normal simulation. This will be address when we add the aerosol interface.